### PR TITLE
Testing/9/tests for lcss matcher

### PR DESCRIPTION
**Pull request by zenon18**
Added the following unit tests from issue #9:
merge
compress
same_trajectory_scheme
find_stationary_points
drop_stationary_points

add_matches_for_stationary_points will be done in a separate branch and pull request.
The testing/9/tests-for-LCSSMather can be deleted after this pr

### CODE REVIEW

File: tests/test_lcss_compress.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase
+
+from mappymatch.matchers.lcss.ops import compress
```
Comment (nreinicke) [inline tests/test_lcss_compress.py:3]: I got an error on this line; should we be importing compress from the `mappymatch.matchers.lcss.utils` module instead?

**Review by nreinicke — COMMENTED** [review]
This is really good stuff, thanks for adding all of these tests in.

I got an error when attempting to run the test suite (see comment). When I updated the import to `mappymatch.matches.lcss.utils` all the tests passed for me so should be a quick fix.

File: tests/test_lcss_compress.py (hunk 1 above)
Comment (zenon18) [reply tests/test_lcss_compress.py]: You are right. I didn't notice that the import error on the test in the vscode testing framework puts them into a different location. The bug has been fixed in the latest commit.

**Review by nreinicke — APPROVED** [review]
This looks good, thank you!

**Pull request merged**